_dev/src/store/modules/paypal/actions.js

```diff
@@ -43,4 +43,13 @@ export default {
       return Promise.resolve();
     });
   },
+  updatePaypalStatus({commit, getters}) {
```

Comment:
By using `update`, I understand that you're trying to send data to store to a controller. `refreshPaypalStatus` seems more appropriate.

Reply:
Renamed to refreshPaypalStatus() for a better understanding.